enterprise/views.py

```diff
@@ -633,53 +637,80 @@ def get_enterprise_course_enrollment_page(
         Render enterprise specific course track selection page.
         """
         platform_name = get_configuration_value('PLATFORM_NAME', settings.PLATFORM_NAME)
+
+        course_title = ''
+        course_short_description = ''
+        course_pacing = ''
         course_start_date = ''
-        if course_run['start']:
-            course_start_date = parse(course_run['start']).strftime('%B %d, %Y')
-
-        # Format the course effort string using the min/max effort fields for the course run.
-        course_effort = ungettext_min_max(
-            '{} hour per week',
-            '{} hours per week',
-            '{}-{} hours per week',
-            course_run['min_effort'] or None,
-            course_run['max_effort'] or None,
-        ) or ''
-
-        # Parse course run image.
-        course_run_image = course_run['image'] or {}
-
-        # Retrieve the enterprise-discounted price from ecommerce.
-        course_modes = self.set_final_prices(course_modes, request)
-        premium_modes = [mode for mode in course_modes if mode['premium']]
-
-        # Parse organization name and logo.
+        course_image_uri = ''
+        course_effort = ''
+        course_full_description = ''
+        course_level_type = ''
+        premium_modes = []
+        expected_learning_items = []
+        staff = []
         organization_name = ''
         organization_logo = ''
-        if course['owners']:
-            # The owners key contains the organizations associated with the course.
-            # We pick the first one in the list here to meet UX requirements.
-            organization = course['owners'][0]
-            organization_name = organization['name']
-            organization_logo = organization['logo_image_url']
-
-        # Add a message to the message display queue if the learner
-        # has gone through the data sharing consent flow and declined
-        # to give data sharing consent.
-        if enterprise_course_enrollment and not data_sharing_consent.granted:
-            messages.add_consent_declined_message(request, enterprise_customer, course_run.get('title', ''))
-
-        if not is_course_run_enrollable(course_run):
-            messages.add_unenrollable_item_message(request, 'course')
-            course['enrollable'] = False
+        course_and_course_run_available = False
+        course_enrollable = True
+        if course and course_run:
+            course_and_course_run_available = True
+            expected_learning_items = course['expected_learning_items']
+            # Parse organization name and logo.
+            if course['owners']:
+                # The owners key contains the organizations associated with the course.
+                # We pick the first one in the list here to meet UX requirements.
+                organization = course['owners'][0]
```

Comment:
nit: [:1]

Reply:
This statement is protected by the condition `if course['owners']` so no `IndexError` will be raised and also we need an individual item for organization not a list.